### DISCUSSION TITLE
Self signed certificated for http metrics

### DISCUSSION THREAD

**Discussion by Rodrigo010497** [jthomperoo/custom-pod-autoscaler#176] [General]
Hello I am just wondering how do I  use the http metric with an endpoint which has a self signed certificate is there anyway to skip the ssl verification like curl --insecure ?

**Comment by jthomperoo** [comment]
Hi, there's a way you can reference the cert if you are able to load it into the container, see the docs here for how to configure that:
https://custom-pod-autoscaler.readthedocs.io/en/stable/user-guide/methods/#http

There isn't currently a way to skip cert checking, but that might be a nice new feature if that would help with development - if you'd like that added could you create an issue for it? Thanks!
